docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-modify-proc.adoc

```diff
@@ -0,0 +1,45 @@
+[id='environment-authoring-ha-modify-proc']
+= Modifying the template for the High Availability authoring environment
+
+By default, the high-availability authoring template creates a MySQL pod to provide the database server for the {KIE_SERVER}. If you prefer to use PostgreSQL or to use an external server (outside the OpenShift project), you need to modify the template before deploying the environment.
+
+You can also modify the High Availability authoring template to change the number of replicas initially created for {CENTRAL}.
+
+.Procedure
+
+Edit the `rhpam-70-authoring-ha.yaml` template file to make any of the following changes as necessary.
+
+* If you want to use PostgreSQL instead of MySQL, replace several sections of the file, marked with comments, with sections from the `rhpam70-kieserver-postgresql.yaml` file that are also marked with comments. All serctions are marked from `BEGIN` to `END`:
+
+. Replace the section named `MySQL database parameters` with the section named `PosgrreSQL database parameters`. (Take this section and all subsequent replacement sections from the `rhpam70-kieserver-postgresql.yaml` file.)
+. Replace the section named `MySQL service` with the section named `PosgrreSQL service`.
+. Replace the section named `MySQL driver settings` with the section named `PosgreSQL driver settings`.
+. Replace the section named `MySQL deployment config` with the section named `PosgreSQL deployment config`.
+. Replace the section named `MySQL persistent volume claim` with the section named `PosgreSQL persistent volume claim`.
+* If you want to use an external database server:
+. Remove the following sections of the file, marked by comments from `BEGIN` to `END`:
++
+** `MySQL service`
+** `MySQL deployment config`
+** `MySQL persistent volume claim`
++
+. In the section named `MySQL driver settings` change the following values:
++
+** `RHPAM_DRIVER` to the driver for your database server. The following drivers are available:
++
+*** `mysql`
+*** `postgresql`
+*** ???
+** `RHPAM_SERVICE_HOST` to the host name of your database server
+** `RHPAM_SERVICE_PORT` to the port of your database server
+** `KIE_SERVER_PERSISTENCE_DIALECT` to the dialect for your database server. The following dialects are available:
++
+*** `org.hibernate.dialect.MySQLDialect`
+*** `org.hibernate.dialect.PostgreSQLDialect`
+*** ???
+* If you want to change the number of replicas initially created for {CENTRAL}:
+. Find the comment line `## Replicas for Business Central`.
```

Comment:
These should technically be steps/ordered since they are sequential actions, but that would look odd and inconsistent. I would just consolidate as one bulleted item anyway, but maybe you think that's worse.  As " Find the comment line `## Replicas for Business Central` and on the next line, change the number of replicas to the desired value." Or maybe, "On the line following the `## Replicas for Business Central` comment, change ...". Just a thought.

Reply:
They actually are steps in the source, but asciidoctor as we use it fails to format them as steps.